docs/pages/upgrading/agent-managed-updates/bootstrap-ec2-agent.mdx

```diff
@@ -0,0 +1,333 @@
+agents in the `development` group are upgraded, the next group, `staging`, begins the upgrade process with
+a 24-hour delay at 05:00 UTC.
+
+If any agent in the `development` group fails, the upgrade process must stop. This behavior is controlled by
+the `halt-on-error` strategy value.
+
+Run the following command to create or update the resource:
+
+```code
+$ tctl create -f autoupdate_config.yaml
+```
+
+## Creating join tokens
+
+Joining a Teleport cluster is the act of establishing trust between a new Teleport instance and all
+the existing instances already part of the Teleport cluster. At the end of the joining process,
+the Auth Service signs certificates for the joining instance. Those certificates represent the
+trust that was established.
+
+For bootstrapping EC2, the most appropriate methods are creating short-lived or long-lived tokens,
+or using delegated join with the IAM join method.
+
+Create a join token with an explicit TTL and the right service type.
+By default, `tctl tokens add` issues short-lived tokens (expiry is 30 minutes if you omit `--ttl` parameter).
+
+```code
+$ tctl tokens add --type=node --ttl=1h
+The invite token: <Var name="place.your.join.token-xxxx" />
+This token will expire in 60 minutes.
+
+Run this on the new node to join the cluster:
+
+> teleport start \
+   --roles=node \
+   --token=<Var name="place.your.join.token-xxxx" /> \
+   --ca-pin=sha256:87c0ab83fb28159d33905fa604093eb0967b4d397d57a9e05113e7f9502af485 \
+   --auth-server=example.teleport.sh:3025
+
+Please note:
+
+  - This invitation token will expire in 60 minutes
+  - example.teleport.sh:3025 must be reachable from the new node
+```
+
+After executing this command, we should receive generated join token name that will be used in the following steps
+of the cloud-init script to generate the Teleport configuration file.
+
+If you need something that lasts longer than a few hours, prefer to create a token resource with a concrete
+`metadata.expires` timestamp instead of cranking `--ttl` to very large values.
+
+Example (set an appropriate, bounded expiry):
+
+```yaml
+# token-long-lived.yaml
+kind: token
+version: v2
+metadata:
+  name: node-join-end-of-year
+  expires: 2025-12-31T23:59:00Z
+spec:
+  join_method: token
+  roles: [Node]
+```
+
+```code
+$ tctl create -f token-long-lived.yaml
+provision_token "node-join-end-of-year" has been created
+```
+
+With the token created by the resource configuration, `node-join-end-of-year` is the secret join token
+name that must be used in the next steps with the cloud-init script, instead of the auto-generated token
+created by the `tctl tokens add` command (<Var name="place.your.join.token-xxxx" />).
+
+<Admonition type="note">
+    To create a delegated token that allows a cluster to be joined via IAM, read the full guide and examples here:
+    [Joining Services via AWS IAM Role](../../enroll-resources/agents/aws-iam.mdx).
+</Admonition>
+
+## Create an EC2 Instance
+
```

Comment:
Thank you for providing both UI and CLI instructions, this is very valuable.

I think tabs would be more readable here, but maybe check with @ptgott before doing this change as they might not agree.